tests/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+enable_testing()
+find_package(GTest 1.10 REQUIRED)
+
+# Unit tests
+add_executable(cps-tests loader.cpp utils.cpp version.cpp)
+target_link_libraries(cps-tests PRIVATE libcps)
+target_link_libraries(cps-tests PRIVATE GTest::gtest GTest::gtest_main)
+
+include(GoogleTest)
+gtest_discover_tests(cps-tests)
+
+# Integration tests
+find_package(
+  Python 3.11
+  COMPONENTS Interpreter
+  REQUIRED)
```

```suggestion
  REQUIRED
)
```

Just for consistent paren placement.